src/account/AccountStorage.sol

```diff
@@ -51,8 +51,7 @@ struct AccountStorage {
     mapping(address => PluginData) pluginData;
     // Execution functions and their associated functions
     mapping(bytes4 => SelectorData) selectorData;
-    mapping(FunctionReference validationFunction => ValidationData) validationData;
-    mapping(address caller => mapping(bytes4 selector => bool)) callPermitted;
+    mapping(FunctionReference => ValidationData) validationData;
```

Comment:
We don't need to remove the named mapping param `validationFunction`

Reply:
Ah good catch! Think was from one of the rebases, will revert